### DISCUSSION TITLE
Add `crs` option to `/info` endpoints

### DISCUSSION THREAD

**Discussion by JinIgarashi** [developmentseed/titiler#1063] [General]
The latest titiler now no longer returns lng lat bounds from `/info` endpoint. Instead, I have to use `/bounds` endpoint with `crs=EPSG:4326` option to fetch lng/lat coordinates.

I am not against new endpoints return original coordinates systems's bounds, but it would be better if `/info` can have `crs` option, so we can get all necessary information from them other than requesting to another bounds endpoint.

**Comment by vincentsarago** [comment]
@JinIgarashi Adding crs option to `/info` might be a bit complex. the `info` response comes from rio-tiler `.info()` method which doesn't take any option (except `indexes=` for stac reader).

So either we make change to rio-tiler and add a new `crs` option or add the option only in titiler to transform the bounds received from rio-tiler method. In both case I'll be a bit sceptical TBH. 

Can I ask why you need this? The bounds in the geographic crs can be found in the `tilejson` or `tilesets` responses

**Comment by JinIgarashi** [comment]
@vincentsarago I am using bounds in lng/lat format in maplibre, previously info endpoint was enough to fetch all information for maplibre, but now I need to use both info and bounds endpoints. If titiler still can return lnglat bounds from info endpoint like what v0.18 did, it can be easier to integrate with map library.

by the way, this `/info` endpoint breading change is missing in your [change log](https://github.com/developmentseed/titiler/blob/main/CHANGES.md).

**Reply by vincentsarago** [reply]
> by the way, this /info endpoint breading change is missing in your [change log](https://github.com/developmentseed/titiler/blob/main/CHANGES.md).

Yeah, it's pretty hard to map all the breaking change happening in rio-tiler to titiler 😬 

In the *new* `/info` response you'll also be missing the `min/max` zoom information, which will only be available through the `tilejson` endpoint. 

If you really want the `bounds` in geographic CRS (epsg:4326) you can you the `/info.geojson` endpoint https://github.com/developmentseed/titiler/blob/6fcd564dc456b420af7ec5f74c991e007fee714f/src/titiler/core/titiler/core/factory.py#L388-L405 which is how we do in the cog viewer extension https://github.com/developmentseed/titiler/blob/main/src/titiler/extensions/titiler/extensions/viewer.py#L36

### Summary

- `/info` endpoint now return the dataset CRS and dataset bounds (in the dataset's CRS)
- `/info.geojson` does have the bounds in geographic CRS (but not min/max zoom)
- `/info` endpoint doesn't have `min/max` zoom metadata, so it's better to use `/{tileMatrixSetId}/tilejson.json` or `/{tileMatrixSetId}/tiles` endpoints
